Fix typo (#13494)
dotnet/docs: docs/machine-learning/resources/tasks.md

### Binary classification inputs and outputs

> For best results with binary classification, the training data should be balanced (i.e. equal numbers of positive and negative training data). Missing values should be handled before training.

Reviewer: ```suggestion
For best results with binary classification, the training data should be balanced (that is, equal numbers of positive and negative training data). Missing values should be handled before training.
```
